apps/web/src/common/modules/widgets/_widget-fields/WidgetFieldTableDataField.vue

```diff
@@ -32,16 +32,20 @@ const state = reactive({
     ]),
     selectedFieldType: 'dynamicField',
     selectedItem: undefined as undefined | MenuItem[] | string,
+    selectedCriteria: undefined as undefined | MenuItem[] | string,
     multiSelectable: computed(() => state.selectedFieldType === 'staticField'),
     menuItems: computed<MenuItem[]>(() => {
-        const dataTarget = state.multiSelectable ? 'data_info' : 'labels_info';
         if (!props.dataTable) return [];
-        const dataInfoList = Object.keys(props.dataTable[dataTarget] ?? {}) ?? [];
-        return dataInfoList.map((d) => ({
-            name: d,
-            label: d,
-        }));
+        return state.selectedFieldType === 'dynamicField' ? state.labelInfoMenuItems : state.dataInfoMenuItems;
     }),
+    dataInfoMenuItems: computed<MenuItem[]>(() => (Object.keys(props.dataTable?.data_info ?? {}) ?? []).map((d) => ({
```

Comment:
nitpick: you don't need `?? []` here. `Object.keys(props.dataTable?.data_info ?? {}` is always array.